MANIFEST.in

```diff
@@ -24,6 +24,9 @@ recursive-include lib/ansible/module_utils/powershell *.psm1
 recursive-include lib/ansible/modules/windows *.ps1
 recursive-include lib/ansible/galaxy/data *.yml *.j2 README.md ansible.cfg inventory .git_keep
 recursive-include lib/ansible/config *.yml
+recursive-include lib/ansible/modules *.yml
```

Comment:
Since the sanity test accepts both .yml and .yaml, this should too.

Reply:
rule in core is not to have .yaml files so that should not be needed, but we allow users to add .yaml to their own content.